react/components/EXPERIMENTAL_Select/Option.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import OptionProps from 'react-select'
+
+import IconCheck from '../icon/Check'
+
+const Option = (props: OptionProps) => {
+  const {
+    children,
+    getStyles,
+    isMulti,
+    innerProps,
+    innerRef,
+    isSelected,
+  } = props
+  return (
+    <div style={getStyles('option', props)} ref={innerRef} {...innerProps}>
+      <div className="flex">
+        {isMulti ? null : (
+          <span className="flex self-center pr3" style={{ width: '1rem' }}>
+            {isSelected ? <IconCheck /> : null}
+          </span>
```

Comment:
Why not
```suggestion
        {!isMulti && isSelected && (
          <span className="flex self-center pr3" style={{ width: '1rem' }}>
            <IconCheck />
          </span>
```

And add 1rem padding/margin to `children`?

Reply:
Hmmm there's a small detail that makes this not to work. The scenario is:
* multi select: i don't want anything but `{children}` 
* single select: if the option is selected, I need the check icon. Else, I need this span to have the exact same width in order to keep all options aligned in the list. Since the minimum width we have on tachyons is of 10% (not fixed btw), we cannot achieve this result. Using this fixed width as inline style avoids any possible unalignment between options.

Does it make sense?

Reply:
Yes, make sense!